lib/bigintmath.ts

```diff
@@ -11,14 +11,18 @@ export const MILLION = JSBI.BigInt(1e6);
 export const BILLION = JSBI.BigInt(1e9);
 const HOUR_SECONDS = 3600;
 export const HOUR_NANOS = JSBI.multiply(JSBI.BigInt(HOUR_SECONDS), BILLION);
-export const MINUTE_NANOS = JSBI.multiply(SIXTY, BILLION);
+export const MINUTE_NANOS_JSBI = JSBI.multiply(SIXTY, BILLION);
```

Comment:
One minor suggestion: should we call these constants `*_BIG` instead of `*_JSBI`?  Then when we finally rip JSBI out and replace it with normal `bigint` then we won't have to update all uses of these constants?

Also fine to leave as-is.

Reply:
I'm hoping that most of these will still go away (or at least not need to be exported) when we're more up to date, so I'll leave it as is for now.